xsens-tf-bridge/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 2.8.9)
+find_package(YARP REQUIRED)
+include_directories(${YARP_INCLUDE_DIRS})
```

Comment:
Please use `target_include_directories` on the specific target for which you want to set the include directories, rather then `include_directories.